common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,14 +244,21 @@ class ModuleI18nService(object):
     i18n service.
 
     """
+    log.info('************* ModuleI18nService Usage!!!! **********')
 
     def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
+        msg = '************* ModuleI18nService getattr call!!! Attr: {0} ************'.format(name)
+        log.info(msg)
+        try:
+            return getattr(xblock_translation, name)
+        except:
+            return getattr(django.utils.translation, name)
```

Comment:
Are you going to need to have more specific fallbacks than this? In particular, don't you have to check whether this *specific* xblock has translations, rather than just whether the utility library has a translation function?